anabot/config/__init__.py

```diff
@@ -20,20 +21,30 @@
     'profile_name': None,
 }
 
+# From python3.12 RawConfigParser doesn't have readfp method, we need to use read_file instead for RHEL 10 and Fedora 40
+def use_read_file():
+    return is_distro_version_ge('rhel', 10) or is_distro_version_ge('fedora', 40)
```

Comment:
Since the method has been removed in python 3.12, you can check the python version rather than the distro version:
`sys.version_info`

Reply:
Thanks for the alternative, I agree with your change.